discovery/consul/consul.go

```diff
@@ -40,7 +41,11 @@ func (s *ConsulDiscoveryService) Initialize(uris string, heartbeat int) error {
 		return err
 	}
 	s.client = client
-	s.heartbeat = time.Duration(heartbeat) * time.Second
+	if heartbeat < 1 {
```

Comment:
the check is already done in `manage.go` and `join.go` I don't think it's necessary to do it here. (it's a lot of duplicated code in every discovery service)